ship.sh

```diff
@@ -86,14 +96,15 @@ function ship_ps() {
 # }
 
 function ship_rm() {
+    check_root
```

Comment:
Thanks @omegazyadav, I tested it and it looks good to me. Only minor thing is could you fix this indentation?  in `ship_rm()` function.

Reply:
@rbalman This has been addressed. Thank you.